sandbox.js

```diff
@@ -9,6 +9,13 @@ let c2paIsLoading = false;
 const messageQueue = [];
 let isProcessing = false;
 const manifestMap = {};
+let computeDataURL;
+
+window.addEventListener('message', (event) => {
+  if (event.data.type === 'valueFromSettings') {
+    computeDataURL = event.data.data;
```

Comment:
wouldn't it make more sense to have an object for the settings? In case we add more options later.
I would do smthg like event.[..].computeDataURL